client/web/src/search/home/SearchPage.tsx

```diff
@@ -98,9 +96,9 @@ export const SearchPage: React.FunctionComponent<SearchPageProps> = props => {
                 />
             </div>
             <div className="flex-grow-1">
-                {props.isSourcegraphDotCom &&
-                    props.showRepogroupHomepage &&
-                    (!props.authenticatedUser || !props.showEnterpriseHomePanels) && <LoggedOutHomepage {...props} />}
+                {props.isSourcegraphDotCom && (!props.authenticatedUser || !props.showEnterpriseHomePanels) && (
```

Comment:
I don't think we should show the logged out homepage if the user is logged in but home panels are disabled. We should show an empty page instead.

```suggestion
                {props.isSourcegraphDotCom && (!props.authenticatedUser) && (
```